Stripe/STPTheme.m

```diff
@@ -60,6 +61,8 @@ - (instancetype)init {
         _errorColor = STPThemeDefaultErrorColor;
         _font = STPThemeDefaultFont;
         _emphasisFont = STPThemeDefaultMediumFont;
+        _translucentNavigationBar = NO;
+        _barStyle = -1;
```

Comment:
I'm using -1 as a sentinel value: if the user hasn't set `barStyle`, we use the computed default value, otherwise we use their chosen value. I'm open to other suggestions for accomplishing the same behavior – maybe an internal boxed enum would be better?

```
- (UIBarStyle)barStyle {
    UIBarStyle defaultStyle = [self barStyleForColor:self.primaryBackgroundColor];
    if (_barStyle < 0) {
        return defaultStyle;
    }
    else {
        return _barStyle;
    }
}
```